portal/server/test-mocks/mock-portal-session-user.ts

```diff
@@ -0,0 +1,17 @@
+import { PortalSessionUser } from '../types/portal-session-user';
+import { ROLES } from '../constants';
+
+export const MOCK_PORTAL_SESSION_USER: PortalSessionUser = {
+  _id: '65954cc526d3899694cafff2',
+  username: 'PAYMENT_REPORT_OFFICER',
+  firstname: 'PAYMENT',
+  surname: 'REPORT OFFICER',
+  email: 'test@testing.com',
+  roles: [ROLES.PAYMENT_REPORT_OFFICER],
```

Comment:
You should be able to just use `'PAYMENT_REPORT_OFFICER'` here rather than using the constant (it would be good to stop using constants where the types can handle that for us)